structure/0048-communications-team.md

```diff
@@ -0,0 +1,70 @@
+# Communications Team
+
+This document has been made according to the process outlined in [RFC
+0006][RFC6].
+
+## Summary
+
+The Communications Team is the team in charge of communications within and
+outside the Quilt Project. This team is also responsible for working with the
+development teams to communicate the current status of development progress
+with the community and with any partners. In essence, it may be thought of as
+a "public relations department" of sorts.
+
+## Motivation
+
+This team is needed because developers would rather focus on development than
+on public relations, status updates, or team to-do lists. Historically, this
+has been managed by the Community Team. However, that team's scope is already
+wide enough. Communications fall outside the realm of the Quilt community, and
+thus this should have its own separate team.
+
+## Explanation
+
+The Communications Team will be responsible for all communications outside of
+and within the Quilt Project. This is difficult to define without providing
+specific examples, which include:
+
+- writing and publishing blog posts
+- managing the official QuiltMC social medias, with the exception of Discord
+- managing relations with other platforms and communities outside of a
+  moderation context
+- writing and publishing general status updates on the progress and
+  development of Quilt
+- assisting in management of public development boards and to-do lists,
+  including GitHub project boards
+- managing the content of the website, not including the functionality
```

Comment:
Would this mean that the Communications Team are in charge of framing content that is currently covered by the Community Team such as Code of Conduct? (This is at least how I would read it)

Reply:
It was my impression that this would specifically be stuff like the proposed user wiki, but I suppose that if we were to make it a subteam of the Community Team (as seemed to be the consensus during our dev meeting yesterday), it could also include the code of conduct and the other things currently on the website.
CC @CheaterCodes, I feel like he would be able to answer this better than I, as this was his original idea

Reply:
I would think that a user wiki would be moderated by the community team. I think "content" is more like a generalization of the current blog posts.

Reply:
I think it is fairly obvious that the comm team will have to make sure the content of the website matches the expectation. In the case of the CoC, it has to be decided and written by the relevant persons with the help of the community.

Maybe the wording could be changed to make it more obvious that the team is responsible for administrating the content, and only write what's relevant to them? I assume in a few cases the comm team would just take a specific messages and put them onto the website.

Reply:
Which team do you mean with "comm team"

Reply:
... right, communication team haha

Reply:
> but I suppose that if we were to make it a subteam of the Community Team (as seemed to be the consensus during our dev meeting yesterday

This has been done. To reduce confusion (comm team or comm team?), it's also been renamed to the Outreach Team, thanks for the idea <@902116671999860757>.

Reply:
To answer the original question:
The content of the Code of Conduct would not be the responsibility of the outreach team.
Putting it on the website would be,
There's some details here about where it would be in it's original form (and I'd argue that should be an RFC on github).